components/dialog/DialogDownloadNotAllowed.vue

```diff
@@ -0,0 +1,46 @@
+<script setup lang="ts">
+defineProps<{
+  show: boolean;
+}>();
+defineEmits<{
+  close: [];
+}>();
+
+const runtimeConfig = useRuntimeConfig();
+const email = runtimeConfig.public.mediaSupportEmail;
+const contact = () => {
+  window.location.href = `mailto:${email}`;
+};
+</script>
+
+<template>
+  <DialogBase
+    :show="show"
+    :title="$t('download.download-track')"
+    class="text-center"
+    @close="$emit('close')"
+  >
+    <div class="flex max-w-xl flex-col items-center justify-center">
+      <NuxtIcon name="download" class="mt-5 text-center text-5xl" />
+      <h3 class="my-3 text-2xl font-extrabold leading-tight text-label-1">
+        {{ $t("download.messages-not-allowed.title") }}
+      </h3>
+
+      <p class="px-10 text-label-2">
+        {{ $t("download.messages-not-allowed.description") }}
+      </p>
+      <TooltipBase placement="top" :offset="8" class="mb-3 mt-5">
+        <ButtonStyled intent="primary" @click="contact">
```

Comment:
Isn't it better to use an `a` tag here, since it basically is a navigation?

Reply:
true but i'd rather use <a> for simplicity